src/main.rs

```diff
@@ -93,9 +99,10 @@ fn read_stdin() -> Result<String, Box<dyn std::error::Error + Send + Sync>> {
     Ok(buffer)
 }
 
-fn format(url: &str, stdin: &str) -> Result<String, BlackdError> {
+fn format(url: &str, line_length: &i32, stdin: &str) -> Result<String, BlackdError> {
     let resp = minreq::post(url)
         .with_header("X-Fast-Or-Safe", "fast")
+        .with_header("X-Line-Length", line_length.to_string())
```

Comment:
on second thought I would actually prefer if we only set this header if the user overrides line length. Otherwise we can just let the blackd server decide the default.

Reply:
Updated.